cli/up.go

```diff
@@ -411,7 +419,12 @@ func streamHandler(stream network.Stream) {
 				return
 			}
 		}
-		stream.SetWriteDeadline(time.Now().Add(25 * time.Second))
-		tunDev.Iface.Write(packet[:size])
+		err = stream.SetWriteDeadline(time.Now().Add(25 * time.Second))
+		if err != nil {
+			fmt.Println("[!] Failed to set write deadline: " + err.Error())
+			stream.Close()
+			return
+		}
+		_, _ = tunDev.Iface.Write(packet[:size])
```

Comment:
Is no error checks intended here?

Reply:
I'm fine with ignoring the error for now. I'm working on https://github.com/hyprspace/hyprspace/issues/16 right now, in which I'll clean up the error handling everywhere as well.